docs/diagrams/FindSequenceDiagram.puml

```diff
@@ -34,11 +31,11 @@ deactivate FindCommandParser
 AddressBookParser -> FindCommandParser : parse("XYZ")
 activate FindCommandParser
 
-create Person
-FindCommandParser --> Person
-activate Person
-Person --> FindCommandParser : person
-deactivate Person
+create NameContainsKeywordsPredicate
+FindCommandParser --> NameContainsKeywordsPredicate
```

Comment:
To be consistent, you can add in the constructor call when creating an object in the sequence diagram.

```suggestion
FindCommandParser --> NameContainsKeywordsPredicate : NameContainsKeywordsPredicate(arguments here)
```